RubeesEat/Model/InMemoryBillRepository.cs

```diff
@@ -39,31 +66,19 @@ public IReadOnlyList<Bill> GetAllForUser(Person user)
                      .ToList();
     }
 
-    public IReadOnlyList<BalanceChange> GetRecentBalanceChanges(Person user, int amount)
-    {
-        var billsWithUser = GetAllForUser(user).OrderByDescending(bill => bill.Date);
-        var balanceChanges = new List<BalanceChange>();
-        int counter = 0;
-        foreach (var bill in billsWithUser)
-        {
-            if (counter >= amount)
-                break;
-            var sum = bill.EntryLines
-                          .Where(entryLine => entryLine.Person.Id == user.Id)
-                          .Sum(entryLine => entryLine.Amount);
-
-            balanceChanges.Add(new BalanceChange(sum, bill.Date, bill.Description, bill.Id));
-            counter++;
-        }
-
-        return balanceChanges;
-    }
-
     public decimal GetBalance(Person user)
     {
         return _bills.Values
                      .SelectMany(e => e.EntryLines)
                      .Where(e => e.Person == user)
                      .Sum(e => e.Amount);
     }
+
+    private BalanceChange GetBalanceChangesForUser(Bill bill, Person currentUser)
```

Comment:
Please convert this to a static local method at the bottom of the method it is used in.